alert-policies/apache-flink/ShuffleNettyUsedMemory.yml

```diff
@@ -0,0 +1,26 @@
+name: Shuffle Netty Used Memory
+
+description: |+
+  This alert is triggered when the shuffle netty - used memory is higher than 2mb during 5 minutes.
```

Comment:
I feel there is no need of using '-' in the text. Request you to kindly remove it.

Reply:
Required changes are done.